apps/emqx_mix_utils/lib/mix/tasks/emqx.eunit.ex

```diff
@@ -142,8 +142,23 @@ defmodule Mix.Tasks.Emqx.Eunit do
   end
 
   defp discover_tests(%{cases: [], modules: []} = _opts) do
+    ## `eunit` is buggy here: the docs says that `{application, _}` uses the application's
```

Comment:
It's perhaps not a bug for eunit.
In rebar3 convention, test modules reside in `test` dir, but not `ebin`, and test modules are not added to the `modules` list in `.app`

Reply:
still claimed it being a 'bug' in the following lines and the PR description.

Reply:
IMO, it still is... the docs say it uses the module list provided explicitly or from `.app`, though that's not entirely true, since it unconditionally adds `mod_tests` (if it exists) for each `mod`, be it in the module list or not.